### PR TITLE
add terraform-tools to the docker images

### LINKED ISSUE #185: Include terraform-tools in the generated docker images
### TL;DR

Include terraform-tools in the generated docker images

### Terraform Resources

_No response_

### Detailed design

```markdown
use


/builder/google-cloud-sdk/bin/gcloud -q components install terraform-tools 
```

or 

```
apt-get -y install google-cloud-sdk-terraform-tools
```

depending on the base image
```


### Additional information

_No response_

### PR DESCRIPTION
**Pull request by daniel-cit**
Fixes #185

### CODE REVIEW

File: modules/cloudbuild/cloudbuild_builder/Dockerfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ ENV ENV_TERRAFORM_VERSION=$TERRAFORM_VERSION
 ENV ENV_TERRAFORM_VERSION_SHA256SUM=$TERRAFORM_VERSION_SHA256SUM
 
 RUN apt-get update && \
+    apt-get -y install google-cloud-sdk-terraform-tools && \
     apt-get -y install curl jq unzip git ca-certificates && \
```
Comment (apeabody) [inline modules/cloudbuild/cloudbuild_builder/Dockerfile:27]: I'd suggest doing as a single apt-get transaction.

```suggestion
    apt-get -y install curl jq unzip git ca-certificates google-cloud-sdk-terraform-tools && \
```

**Comment by comment-bot-dev** [comment]
@daniel-cit
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/cloudbuild/cloudbuild_builder/Dockerfile (hunk 1 above)
Comment (daniel-cit) [reply modules/cloudbuild/cloudbuild_builder/Dockerfile]: great idea.
done

**Comment by comment-bot-dev** [comment]
@daniel-cit
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by bharathkkb — COMMENTED** [review]
Lets also add to the older cloudbuild module?
https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/modules/cloudbuild/cloudbuild_builder/Dockerfile

**Comment by daniel-cit** [comment]
> Lets also add to the older cloudbuild module? https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/modules/cloudbuild/cloudbuild_builder/Dockerfile

@bharathkkb it has also ben added. 

In the older module it is a bit different, it uses `google-cloud-sdk-terraform-tools` instead of  just `terraform-tools` because they use different base images

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**